app/templates/buyers/brief_responses.html

```diff
@@ -12,20 +12,22 @@
         {
             "link": "/buyers",
             "label": "Your Requirements"
+        },
+        {
+            "link": url_for('buyers.view_brief_summary', framework_slug=framework.slug, lot_slug=lot.slug, brief_id=brief_data.id),
+            "label": brief_data.title
         }
         ]
     %}
         {% include "toolkit/breadcrumb.html" %}
     {% endwith %}
 {% endblock %}
 
-{% set responses_count = response_counts['failed'] + response_counts['eligible'] %}
-
 {% block main_content %}
 <div class="grid-row">
     <div class="column-two-thirds">
         {% with
-        heading = ("Responses to &lsquo;" + brief_data.get('title', brief_data['lotName']) + "&rsquo;")|safe,
+        heading = ("Shortlist suppliers")|safe,
```

Comment:
This no longer needs the `safe` filter as it doesn't contain any characters that would be escaped.